CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ In this case you can simply pull down the version you want, and run `docker-comp
 ## [Unreleased]
 
 ### Fixes
+- #696 : actually now uses environment variable SECRET_KEY_BASE. If this was set, unset otherwise you can not login to BETY.
```

Comment:
```suggestion
#696 : breaking change: Now actually uses environment variable SECRET_KEY_BASE, which was silently ignored before. If this was set on your system, unset it or you will not be able to log in to BETY.
```

Maybe too long now? But I think the current wording would make it easy for me to miss that this requires user action if I were reading the changelog without having seen the PR description.

Reply:
My goal is to put this in **BOLD** in the release notes.